src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  width: $calendar-width;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__field {
+    @include field-size($field-width);
+    background-color: #eee;
+    border: $border-field solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
+    transition-delay: 0.5s;
+    transition-timing-function: ease-in-out;
```

Comment:
please define also transition-property with each property that you need, not just 'all' - https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property